pyiron_contrib/atomistic/atomistics/job/simple_atomistic.py

```diff
@@ -0,0 +1,66 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+from pyiron_atomistics.atomistics.job.interactive import GenericInteractive
+import numpy as np
+
+class SimpleAtomistic(GenericInteractive):  # Create a custom job class
```

Comment:
It would be great to add a `to_hdf()` and `from_hdf()` method.

Reply:
When you do this you might find it's convenient to change most (all?) fields to live on an `InputList`.